src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -17,7 +17,7 @@ internal static class NodeConstants
         internal const string NpmStartCommand = "npm start";
         internal const string YarnStartCommand = "yarn run start";
         internal const string YarnCommand = "yarn";
-        internal const string NpmPackageInstallCommand = "npm install";
+        internal const string NpmPackageInstallCommand = "npm install --unsafe-perm";
```

Comment:
Does the test in this PR fail without this flag?

Reply:
as discussed offline